packages/backend/src/migration/1566807284426-CreateTableUserCourseRole.ts

```diff
@@ -4,7 +4,7 @@ export class CreateTableUserCourseRole1566807284426
   implements MigrationInterface {
   public async up(queryRunner: QueryRunner): Promise<any> {
     await queryRunner.query(
-      "CREATE TYPE role_enum AS ENUM ('teacher', 'assistant');",
+      "CREATE TYPE role_enum AS ENUM ('teacher', 'assistant', 'reviewer');",
```

Comment:
Instead of editing this migration in backend v1, please create a new migration in backendv2 that modifies the existing table.